third_party/move/move-vm/runtime/src/interpreter.rs

```diff
@@ -96,12 +100,14 @@ impl Interpreter {
             call_stack: CallStack::new(),
             paranoid_type_checks: loader.vm_config().paranoid_type_checks,
             access_control: AccessControlState::default(),
+            executed_modules: HashSet::new(),
```

Comment:
Nit: the naming "executed_modules" seems a bit weird to me since none of the functions on the call stack has finished execution. Maybe we could name this to something like "active_modules"?